packages/nimble-components/src/wafer-map/modules/event-coordinator.ts

```diff
@@ -38,12 +37,12 @@ export class EventCoordinator {
         event.preventDefault();
     };
 
-    private readonly onMouseMove = (): void => {
-        // TODO HoverHandler - mousemove(e) callback
+    private readonly onMouseMove = (event: MouseEvent): void => {
+        this.hoverHandler.mousemove(event);
     };
 
-    private readonly onMouseOut = (): void => {
-        // TODO HoverHandler - mouseout() callback
+    private readonly onMouseOut = (event: MouseEvent): void => {
+        this.hoverHandler.mouseout(event);
     };
 
     private attachEvents(): void {
```

Comment:
There is still a TODO comment at the bottom of this file related to the hover handler that no longer seems relevant